enforcer-rules/src/test/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -234,7 +234,14 @@ public void testExecute_shouldFailForBadProject() throws RepositoryException {
       verify(mockLog, times(1)).error(errorMessageCaptor.capture());
 
       String errorMessage = errorMessageCaptor.getValue();
-      Truth.assertThat(errorMessage).startsWith("Linkage Checker rule found 112 errors:");
+
+      // Java 11 removed javax.activation package. Therefore the number of expected error differs
+      // between Java 8 and Java 11.
+      // https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1856
+      int expectedErrorCount = System.getProperty("java.version").startsWith("1.8.") ? 112 : 117;
```

Comment:
Java 1.7 is likely more like 112 than 117 but it could be something different entirely.

Reply:
This repository does not support Java 7.

https://github.com/GoogleCloudPlatform/cloud-opensource-java#requirements

Writing this comment, I noticed we don't limit the JDKs to LTS...

Reply:
Followup: https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1859

We have documented this repository works with Java 8.